lib/urldata.h

```diff
@@ -1585,6 +1585,13 @@ enum dupstring {
 
   STRING_COPYPOSTFIELDS,  /* if POST, set the fields' values here */
 
+#ifdef USE_ARES
```

Comment:
For some specific build reasons on some platforms, we avoid ifdef'ing the list of strings and provide them all in all builds, so if you just drop the ifdefs here ...